packages/core/src/client/sitecore-client.ts

```diff
@@ -283,6 +298,21 @@ export class SitecoreClient implements BaseSitecoreClient {
           .join('/')}`;
   }
 
+  /**
+   * Execute a raw GraphQL request using the client's configured GraphQL endpoint(s).
+   * This is a thin pass-through to the underlying {@link GraphQLClient.request},
+   * @param {string | DocumentNode} query GraphQL string or DocumentNode
```

Comment:
The same, please, apply to the interface description defined above
```suggestion
   * @param {string | DocumentNode} query GraphQL query
```